src/y-webrtc.js

```diff
@@ -461,78 +463,85 @@ const openRoom = (doc, provider, name, key) => {
 const publishSignalingMessage = (conn, room, data) => {
   if (room.key) {
     cryptoutils.encryptJson(data, room.key).then(data => {
-      conn.send({ type: 'publish', topic: room.name, data: buffer.toBase64(data) })
+      conn.client.sendToGroup(room.name, buffer.toBase64(data), "text")
     })
   } else {
-    conn.send({ type: 'publish', topic: room.name, data })
+    conn.client.sendToGroup(room.name, data, "json")
   }
 }
 
-export class SignalingConn extends ws.WebsocketClient {
+export class SignalingConn extends Observable {
```

Comment:
This doesn't actually need to extend `Observable` since the client subscribes to events, not this class.